### PR TITLE
slug -> username/slug

### PR DESCRIPTION
**Pull request by mewwts**

### CODE REVIEW

File: pages/loopring/loopring.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+t---
```
Comment (milkyklim) [inline pages/loopring/loopring.md:1]: Typo?

**Review by milkyklim — APPROVED** [review]
All good except one typo.

File: pages/loopring/loopring.md (hunk 1 above)
Comment (mewwts) [reply pages/loopring/loopring.md]: Thanks!

**Review by markusbkoch — APPROVED** [review]
lgtm

**Review by olav — APPROVED** [review]

**Pull request merged**